src/components/ChangePlan/ChangePlan.js

```diff
@@ -23,14 +40,27 @@ const ChangePlan = () => {
         <section className="dp-container">
           <div className="dp-rowflex">
             <div className="col-sm-12" style={{ textAlign: 'center' }}>
-              <h1>Compra un plan</h1>
-              <a href={planUrl} class="dp-button button-medium primary-green">
-                Comprar Plan
-              </a>
-              <br />
-              <br />
-              Información del plan Plan Id {planId} - DiscountId - {safeDiscountId} - PromoCode -{' '}
-              {safePromoId}
+              <Card>
+                <div className="dp-content-plans">
```

Comment:
@GusBaamonde, is this div really necessary? why is this useful?

Reply:
Esa clase los hace todos de la misma altura y presiona los botones en todos los casos para quedar alineados.
Es necesario porque el diseño lo requiere. De lo contrario podría quedar algún plan des balanceado y cuando ampliemos/despleguemos las comparaciones no quedarían en la misma linea. Afecta a la visualización y comparación después (Sé que no es lo mas bonito).

Reply:
Y no podría erscribirse así?

```suggestion
              <Card className="dp-content-plans">
```

Reply:
@GusBaamonde ?

Reply:
changed

Reply:
> changed

No, you did not :P